src/components/Button/style.scss

```diff
@@ -0,0 +1,33 @@
+/* Import Variables and Helpers */
+@import '../../style/helpers';
+
+/* Local Variables */
+$BUTTON_HEIGHT: 60 !default;
+$BUTTON_WIDTH: 280 !default;
+$BUTTON_RADIUS: 5 !default;
```

Comment:
Button radius could be exported to global variables, because it can be shared with other components

Reply:
I left it here for now, If I'll see other parts using same border radius I'll move it to global variables